sys/include/net/ng_netconf.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_netapi Configuration options for network APIs
+ * @ingroup     net
+ * @brief       List of available configuration options for network APIs
+ * @{
+ *
+ * @file
+ * @brief       Definition of global configuration options for netdev and netapi
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NET_CONF_H_
+#define NET_CONF_H_
```

Comment:
why H_H? Too my knowledge we don't have a rule for naming these in RIOT?! And as I used to put the underscore in front (which was in violation of the C standard as I learned), I put one after the name now. Don't think that hurts and it helps to prevent naming clashes